lambda_functions/archive_service/post_comment_on_item.py

```diff
@@ -0,0 +1,42 @@
+import logging
+import json
+from core_layer import helper
+from core_layer.db_handler import Session
+from core_layer.handler import comment_handler, item_handler
+
+
+def post_comment_on_item(event, context = None):
+    """
+    Creates comment on item from archive
+    """
+
+    with Session() as session:
+
+        logger = logging.getLogger()
+        logger.setLevel(logging.INFO)
+
+        try:
+            body = json.loads(event['body']) if isinstance(
+                event['body'], str) else event['body']
+        except:
+            return helper.get_text_response(400, "Malformed request. Please provide a valid request.", event)
+
+        if 'item_id' not in body:
+            return helper.get_text_response(400, "Malformed request. Please provide an item_id.", event)
+
+        try:
+            item = item_handler.get_item_by_id(body['item_id'], session)
+        except:
+            return helper.get_text_response(404, "Item not found", event)
```

Comment:
If we simply try to add the comment with the provided `body['item_id']` the INSERT should fail if no item with the provided id exists.
That would save us one database roundtrip.

Reply:
But we would loose the detailed description on why the call fails, right?

Reply:
I guess that depends on how detailed the traceback of the SQLAlchemy insert is.